### PR TITLE
[zh-CN]: add the translation of <absolute-size> type

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/css/absolute-size/index.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+slug: Web/CSS/absolute-size
+l10n:
+  sourceCommit: 2077d0702d038c9ccc743a53d8ad1c0c21fef5be
+---
+
+{{CSSRef}}
+
+**`<absolute-size>`** [CSS](/zh-CN/docs/Web/CSS) [基本数据类型](/zh-CN/docs/Web/CSS/CSS_Types) 描述了绝对大小关键字。此数据类型用于 {{cssxref("font")}} 简写属性和 {{cssxref("font-size")}} 属性。
+
+字体大小关键字映射到已过时的 HTML `size` 属性，请参阅下面的 [HTML size 属性](#HTML size 属性) 部分。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/absolute-size/index.md:12]:
```suggestion
字体大小关键字映射到已过时的 HTML `size` 属性，请参阅下面的 [HTML size 属性](#HTML size 属性)部分。
```

File: files/zh-cn/web/css/absolute-size/index.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+## 规范
+
+{{Specifications}}
+
+## 参见
+
+- CSS {{cssxref("relative-size")}} 数据类型
+- CSS {{cssxref("font")}} 和 {{cssxref("font-size")}} 属性
+- [CSS 字体](/zh-CN/docs/Web/CSS/CSS_fonts) 模块
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/absolute-size/index.md:132]:
```suggestion
- [CSS 字体](/zh-CN/docs/Web/CSS/CSS_fonts)模块
```

File: files/zh-cn/web/css/absolute-size/index.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+title: <absolute-size>
+slug: Web/CSS/absolute-size
+l10n:
+  sourceCommit: 2077d0702d038c9ccc743a53d8ad1c0c21fef5be
+---
+
+{{CSSRef}}
+
+**`<absolute-size>`** [CSS](/zh-CN/docs/Web/CSS) [基本数据类型](/zh-CN/docs/Web/CSS/CSS_Types) 描述了绝对大小关键字。此数据类型用于 {{cssxref("font")}} 简写属性和 {{cssxref("font-size")}} 属性。
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:10]:
```suggestion
**`<absolute-size>`** [CSS](/zh-CN/docs/Web/CSS) [数据类型](/zh-CN/docs/Web/CSS/CSS_Types)描述了绝对大小关键字。此数据类型用于 {{cssxref("font")}} 简写属性和 {{cssxref("font-size")}} 属性。
```

File: files/zh-cn/web/css/absolute-size/index.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+字体大小关键字映射到已过时的 HTML `size` 属性，请参阅下面的 [HTML size 属性](#HTML size 属性)部分。
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:12]:
```suggestion
字体大小关键字映射到已弃用的 HTML `size` 属性，请参阅下面的 [HTML size 属性](#html_size_属性)部分。
```

File: files/zh-cn/web/css/absolute-size/index.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+  - : 绝对大小：等于 `medium` 字体的 60%，对应已过时的属性 `size="1"`。
+
+- `x-small`
+
+  - : 绝对大小：等于 `medium` 字体的 75%。
+
+- `small`
+
+  - : 绝对大小：等于 `medium` 字体的 89%，对应已过时的属性 `size="2"`。
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:34]: - not consistent with `large` translation below
- "deprecated" => "已弃用"

File: files/zh-cn/web/css/absolute-size/index.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+- `x-large`
+
+  - : 绝对大小：比 `medium` 大 50%，对应已过时的属性 `size="5"`。
+
+- `xx-large`
+
+  - : 绝对大小：等于 `medium` 字体的两倍，对应已过时的属性 `size="6"`。
+
+- `xxx-large`
+  - : 绝对大小：等于 `medium` 字体的三倍，对应已过时的属性 `size="7"`。
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:53]: - "deprecated" => "已弃用"

File: files/zh-cn/web/css/absolute-size/index.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+## 说明
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:55]:
```suggestion
## 描述
```

File: files/zh-cn/web/css/absolute-size/index.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+每个 `<absolute-size>` 关键字值大小是相对于 `medium` 大小和个别设备的特性（例如设备分辨率）而定的。用户代理为每个字体维护一个字体大小表，其中 `<absolute-size>` 关键字是索引。
+
+在 CSS1（1996）中，相邻关键字值索引之间的缩放比例为 1.5，这太大了。在 CSS2（1998）中，相邻关键字值索引之间的缩放比例为 1.2，这对细小的值造成了问题。由于发现相邻绝对大小关键字之间的单一固定比例存在问题，因此不再推荐使用固定比例，为了保持可读性，唯一的建议是最小字体大小不应小于 `9px`。
+
+下表列出了每个 `<absolute-size>` 关键字值的缩放比例、映射到 [`<h1>` to `<h6>`](/zh-CN/docs/Web/HTML/Element/Heading_Elements) 标题以及映射到已过时的 [HTML `size` 属性](#HTML size 属性)。
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:61]:
```suggestion
下表列出了每个 `<absolute-size>` 关键字值的缩放比例、映射到 [`<h1>` to `<h6>`](/zh-CN/docs/Web/HTML/Element/Heading_Elements) 标题以及映射到已弃用的 [HTML `size` 属性](#html_size_属性)。
```

File: files/zh-cn/web/css/absolute-size/index.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+- `x-small`
+  - : 绝对大小：等于 `medium` 字体的 75%。
+- `small`
+  - : 绝对大小：等于 `medium` 字体的 89%，对应已弃用的属性 `size="2"`。
+- `medium`
+  - : 绝对大小：用户首选的字体大小，此值用作参考中等值。对应到属性 `size="3"`。
+- `large`
+  - : 绝对大小：等于 `medium` 字体大 20%，对应已弃用的属性 `size="4"`。
+- `x-large`
+  - : 绝对大小：等于 `medium` 字体大 50%，对应已弃用的属性 `size="5"`。
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:35]:
```suggestion
  - : 绝对大小：比 `medium` 字体大 20%，对应已弃用的属性 `size="4"`。
- `x-large`
  - : 绝对大小：比 `medium` 字体大 50%，对应已弃用的属性 `size="5"`。
```

File: files/zh-cn/web/css/absolute-size/index.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+- `xx-large`
+  - : 绝对大小：等于 `medium` 字体的两倍，对应已弃用的属性 `size="6"`。
+- `xxx-large`
+  - : 绝对大小：等于 `medium` 字体的三倍，对应已弃用的属性 `size="7"`。
+
+## 描述
+
+每个 `<absolute-size>` 关键字值大小是相对于 `medium` 大小和个别设备的特性（例如设备分辨率）而定的。用户代理为每个字体维护一个字体大小表，其中 `<absolute-size>` 关键字是索引。
```
Comment (yin1999) [inline files/zh-cn/web/css/absolute-size/index.md:43]:
```suggestion
每个 `<absolute-size>` 关键字值大小是相对于 `medium` 大小和个别设备的特征（例如设备分辨率）而定的。用户代理为每个字体维护一个字体大小表，其中 `<absolute-size>` 关键字是索引。
```

**Review by yin1999 — APPROVED** [review]
Thanks @T34-active

**Pull request merged**